sdk-lib/src/main/java/cash/z/ecc/android/sdk/SdkSynchronizer.kt

```diff
@@ -379,36 +379,44 @@ class SdkSynchronizer private constructor(
         return !validateAddress(address).isNotValid
     }
 
-    private fun CoroutineScope.onReady() = launch(CoroutineExceptionHandler(::onCriticalError)) {
+    private fun CoroutineScope.onReady() {
         Twig.debug { "Starting synchronizer…" }
 
-        var lastScanTime = 0L
-        processor.onProcessorErrorListener = ::onProcessorError
-        processor.onSetupErrorListener = ::onSetupError
-        processor.onChainErrorListener = ::onChainError
-        processor.state.onEach {
-            when (it) {
-                is Synced -> {
-                    val now = System.currentTimeMillis()
-                    // do a bit of housekeeping and then report synced status
-                    onScanComplete(it.syncedRange, now - lastScanTime)
-                    lastScanTime = now
-                    SYNCED
-                }
+        // Triggering UTXOs fetch at the beginning of the block sync right after the app start, as it makes the
+        // transparent transactions appearance faster
+        launch(CoroutineExceptionHandler(::onCriticalError)) {
+            refreshUtxos()
```

Comment:
Do we also need to call `refreshTransparentBalance()` afterwards?

Reply:
Yes, `refreshTransparentBalance()` should be called. What do you think about calling `refreshTransactions()` after that? It invalidates the storage.

Reply:
Yes, we should call the `refreshTransactions()` as well, as discussed.